### ISSUE TITLE
Craftbook support issue

### ISSUE THREAD

**Issue by idnthvn** [JamesPeters98/ChestsPlusPlus#41]
Hello, Craftbook and chest++ do not like each other .
When using Craftbook pipes, it doesn't allow the use of a chest++ chest. 
https://github.com/EngineHub/CraftBook/issues/1252

Hopefully someway, chest++ can allow Craftbook access? If at all possible.   Thank you

**Comment by JamesPeters98** [comment]
Yeah this will be because Chests++ doesn't use the 'Inventory' of the Chest and so when you right click or access the chest the plugin just opens the linked Inventory for you instead. 

I'm not sure how to overcome issues like this, I need to have a think.

**Comment by JamesPeters98** [comment]
Same as #23

**Comment by idnthvn** [comment]
I am not a coder, but i was trying to think about this. I think what you are saying is. 
So, a hopper allows items to be entered into the chest++ (It is how i transfer between plugin and chest++) Since items being inserted into the chest, that is not a player or hopper is not seen. Is there a listener to catch or see (Item being inserted) into an inventory. Maybe a way to catch an item with a pickup.  Not sure exactly. Was trying to look.   The author of craftbook and author of HopperSorter would most likely be able to help. Those are the two plugins i tried and failed to insert into the chest. 
i do not know how to tag.  -Craftbook Author = me4502

lookes like HopperSorter is helping you https://github.com/LegameMc/ChestsPlusPlus/commit/a004df189575f7b1197e497c81b53e376c1387d2
